templates/network.epp

```diff
@@ -0,0 +1,25 @@
+<%- |
+    Array[Hash] $addresses,
+    String[1] $interface,
+| -%>
+# THIS FILE IS MANAGED BY PUPPET
+# based on https://dn42.dev/howto/wireguard
+[Match]
+Name=<%= $interface %>
+
+[Network]
+DHCP=no
+IPv6AcceptRA=false
+IPForward=yes
+
+# for networkd >= 244 KeepConfiguration stops networkd from
+# removing routes on this interface when restarting
+KeepConfiguration=yes
+<% $addresses.each |$address| { -%>
+
+[Address]
+<% $address.each |$key, $value| { -%>
+<%= $key %>=<%= $value %>
+<% } -%>
+<% } -%>
+
```

Comment:
can you remove the empty line here?

Reply:
but if you change something, this will be git changes on both lines